src/Dropdown.vue

```diff
@@ -48,15 +46,22 @@ export default {
     },
     type: {
       type: String,
-      default: 'default'
+      default: 'light'
     }
   },
   computed: {
     btnType () {
-      return `btn-${this.type}`;
+      if (this.type !== '') {
+        return `btn-${this.type}`;
+      } else {
+        return '';
+      }
     },
     classes () {
-      return [{open: this.showBool, disabled: this.disabledBool}, this.class, this.isLi ? 'dropdown' : this.inInput ? 'input-group-btn': 'btn-group']
+      return [{disabled: this.disabledBool}, this.class, this.isLi ? 'dropdown' : this.inInput ? 'input-group-append': 'btn-group']
```

Comment:
`this.inInput` is part of the **Input Component** which was not migrated, we can trim off some logic here.